packages/common/src/ui/components/NavLink.tsx

```diff
@@ -8,7 +8,7 @@ import { useMatch, Link } from 'react-router-dom';
 const useStyles = makeStyles(theme => ({
   drawerMenuItem: {
     height: 32,
-    margin: '20px 0',
+    marginTop: '20px',
```

Comment:
I read [this](https://mxstbr.com/thoughts/margin/) the other day from another article talking about the old `spacer.gif` .. then playing around with this margin made me think I agree 🤔

Reply:
If it's just a single number, then better to:

```suggestion
    marginTop: 20,
```

Reply:
the margin article was interesting - and good food for thought. I tend to agree - but margins are soooo useful.